src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -150,6 +150,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return;
         }
 
+        $resultMessage = 'Translation files were successfully updated.';
```

Comment:
dot should be removed here as code below can append another string (see L209). Dot should be added at the end if it is not already there.